runThuderbird.sh

```diff
@@ -1,10 +0,0 @@
-#!/bin/bash
```

Comment:
sure, I am fo such directory

Reply:
I was trying to find how can tests be created for thunderbird extensions but so far I have not found anything interesting. So what I do is just run extension and try to click on UI and check results. But then I miss some tests that I have done in earlier development. :( If you find some example, log or tutorial for how to create test for thunderbird extensions I can contribute to writing them.

Reply:
I do the same as you right now. The only thing I can think of is to treat it sort of like a node project, modularize the code and use jest (or similar - mocha+chai or something?). That might also enable us to move to modern javascript using typescript+babel or something? I'm pretty new to javascript but I'm currently doing a typescript reactxp project like this, and ts turns the typescript into javascript, which is then transpiled down to ES5, and jest seems to work well for it? I haven't done it here because I'm not experienced enough to do it quickly but it would be awesome to have it "professionalized" like that